server/routerlicious/packages/lambdas-driver/src/document-router/documentLambda.ts

```diff
@@ -101,6 +112,69 @@ export class DocumentLambda implements IPartitionLambda {
         document.process(message);
     }
 
+    private logMessageDetails(boxcar: IBoxcarMessage, message: IQueuedMessage, documentId: string, tenantId: string) {
```

Comment:
nit: Maybe rename to `getMessageDetails` / `getMessageLumberjackProperties` and write the log inline at line 93 above - Because this function looks more generic to extract message details which can be used for any log statement.